site/docs/v1/tech/apis/_webhook-request-body.adoc

```diff
@@ -32,6 +32,9 @@ include::_event-types.adoc[]
 
 [field]#webhook.global# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`#::
 Whether or not this Webhook is used for all events or just for specific Applications. In almost all cases you want to set this field to `true` and filter on the application Id when processing the webhook.
++
+[since]#Since 1.37.0#
```

Comment:
Is there a better way to phrase or format this?

I was thinking it would be better to say that it's for Tenants and then make the note say that prior to version 1.37.0 it was for Applications, but I didn't see other cases where we did that.